src/components/MiddleChatWindow.js

```diff
@@ -1,6 +1,7 @@
 import { useContext } from "react"
 import "./MiddleChatWindow.css"
 import { ThemeContext } from "../themeContext.js"
+import Message from "./Message"
 import UserLogo from "../assets/sparrow-user-profile.svg"
```

Comment:
Updated new default user-profile image path so as not to revert back to the one that bleeds out when in light-theme mode:
 
```suggestion
import UserLogo from "../assets/user-profile.png"
```

Reply:
Thank you for catching this! :grin: 
I'll commit your suggestion after William updates the PR with the styling. Otherwise he will have to deal with a merge conflict and won't be able to push his code.

Reply:
Sounds good

Reply:
The ternary is gone now too in middle chat window. It's a conditional to account for 3 different names entered.

Reply:
yeah, I think this will definitely show as a merge conflict whenever we try to merge this PR. It should be easy to solve.

Reply:
I think in signin component the conditional has been switched also.

Reply:
this should not touch signin, so it shouldn't be an issue.